src/xsar/sentinel1_dataset.py

```diff
@@ -216,6 +221,8 @@ def __init__(self, dataset_id, resolution=None,
 
         # variables not returned to the user (unless luts=True)
         self._hidden_vars = ['sigma0_lut', 'gamma0_lut', 'noise_lut', 'noise_lut_range', 'noise_lut_azi']
+        # attribute to activate correction on variables, if available
```

Comment:
Just changed the comment, because nothing says at this level that the patch apply to noise.

If in the future we have many variables to patch, and only want to apply patch to some of them, we will have to use a dict like:
```python
self._patch_variable = {
  'noise_lut_azi': True,
  'other_var': False
}
```

But we will handle that later if needed.

Reply:
Oups, I didn't look at your commit, I though it was a review proposal. 
So it is fine for me regarding this comment.